test/datatable.jl

```diff
@@ -342,6 +342,40 @@ module TestDataTable
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), [1]).columns) == [1]
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), 1).columns) == [1]
 
+    @testset "unstack nullable promotion" begin
+        long = DataTable(Any[repeat([:c, :d], inner=12),
+                             vcat(linspace(0, 1, 12), linspace(2, 1, 12)),
+                             repeat(1:12, outer=2), repmat(repeat(1:3, inner=4), 2),
```

Comment:
(Only applies if you keep this.) Simply `repeat(1:3, inner=4, outer=2)`. Same below.